src/config/styles.json

```diff
@@ -80,7 +80,7 @@
   {
     "id": "protomaps-light",
     "title": "Protomaps Light",
-    "url": "https://api.protomaps.com/styles/v2/light.json?key=d828297496b11844",
+    "url": "https://api.protomaps.com/styles/v4/light/$lang.json?key=d828297496b11844",
```

Comment:
I think the common usage is `{lang}` and not `$lang` in maplibre - glyphs, tiles etc use "{}" for parameters.